doc/dev/contributing.md

```diff
@@ -0,0 +1,126 @@
+# Contributing to DAOS
+
+Your contributions are most welcome! There are several good ways to suggest new
+features, offer to add a feature, or just begin a dialog about DAOS:
+
+- Open an issue in [jira](https://jira.hpdd.intel.com)
+
+- Suggest a feature, ask a question, start a discussion, etc. in our [user group](https://daos.groups.io/g/daos)
+
+- Chat with members of the DAOS community real-time on [Gitter](https://gitter.im/daos-storage)
```

Comment:
Seems to have zero content since October, should we be referencing it?

Reply:
yeah, still want to eventually use it.